code/modules/fishing/fish/_fish.dm

```diff
@@ -310,6 +313,9 @@
 
 	SEND_SIGNAL(src, COMSIG_FISH_FORCE_UPDATED, weight_rank, bonus_malus)
 
+	if(force >=15 && hitsound == SFX_DEFAULT_FISH_SLAP) // don't override special attack sounds
+		hitsound = 'sound/creatures/fish/fish_slap2.ogg' // do more damage - do heavier slap sound
```

Comment:
Oh, I've forgot, but you need to also do the opposite at the start of update_fish_force() (and perhaps use a sfx define for it too).

Reply:
oof